test/e2e/run-openrpc-api-test-coverage.ts

```diff
@@ -0,0 +1,644 @@
+import testCoverage from '@open-rpc/test-coverage';
+import { parseOpenRPCDocument } from '@open-rpc/schema-utils-js';
+const mockServer = require('@open-rpc/mock-server/build/index').default;
+import HtmlReporter from '@open-rpc/test-coverage/build/reporters/html-reporter';
+import ExamplesRule from '@open-rpc/test-coverage/build/rules/examples-rule';
+import JsonSchemaFakerRule from '@open-rpc/test-coverage/build/rules/json-schema-faker-rule';
+import paramsToObj from '@open-rpc/test-coverage/build/utils/params-to-obj';
+import {
+  ContentDescriptorObject,
+  ExampleObject,
+  ExamplePairingObject,
+  MethodObject,
+  OpenrpcDocument,
+} from '@open-rpc/meta-schema';
+import openrpcDocument from '@metamask/api-specs';
+const { v4 } = require('uuid');
+
+declare let window: any;
+
+const uuid = v4;
+
+const FixtureBuilder = require('./fixture-builder');
+const {
+  withFixtures,
+  openDapp,
+  unlockWallet,
+  DAPP_URL,
+  WINDOW_TITLES,
+  switchToOrOpenDapp,
+} = require('./helpers');
+
+import { Driver, PAGES } from './webdriver/driver';
+import Rule from '@open-rpc/test-coverage/build/rules/rule';
+import { Call } from '@open-rpc/test-coverage/build/coverage';
+
+const pollForResult = async (driver: Driver, generatedKey: string) => {
+  let result = await driver.executeScript(`return window['${generatedKey}'];`);
+
+  while (result === null) {
+    // Continue polling if result is not set
+    await driver.delay(50);
+    result = await driver.executeScript(`return window['${generatedKey}'];`);
+  }
+
+  // clear the result
+  await driver.executeScript(`delete window['${generatedKey}'];`);
```

Comment:
may be able to save some time by not awaiting this

Reply:
we have to await it or else it could make another driver call and selenium will throw that it cant do multiple things at the same time